synapse/models/inet.py

```diff
@@ -198,7 +198,11 @@ def _normPyStr(self, valu):
 
         try:
             user, fqdn = valu.split('@', 1)
+        except ValueError:
+            mesg = f'Cannot split the email user and fqdn for "{valu}"'
```

Comment:
Maybe something about the missing @ sign?  Something like `Email address expected in <user>@<fqdn> format: {valu}` or something?